ims/database/image.py

```diff
@@ -141,8 +141,11 @@ def fetch_id_with_name_from_project(self, name, project_name):
             image = self.connection.session.query(Image). \
                 filter(Image.project.has(name=project_name)).filter_by(
                 name=name).one_or_none()
-            if image is not None:
-                return image.id
+
```

Comment:
@naved001 You can just add doc_string to the function you tweaked. If there was a doc_string you would have needed to update it anyways. Also follow the pattern in Config Changes PR for the doc_string